virttest/utils_misc.py

```diff
@@ -2240,18 +2240,26 @@ def get_qemu_binary(params):
     Get the path to the qemu binary currently in use.
     """
     # Update LD_LIBRARY_PATH for built libraries (libspice-server)
+    qemu_binary = None
     qemu_binary_path = get_path(_get_backend_dir(params),
                                 params.get("qemu_binary", "qemu"))
 
     if not os.path.isfile(qemu_binary_path):
         logging.debug('Could not find params qemu in %s, searching the '
                       'host PATH for one to use', qemu_binary_path)
-        try:
-            qemu_binary = utils_path.find_command('qemu-kvm')
-            logging.debug('Found %s', qemu_binary)
-        except utils_path.CmdNotFoundError:
-            qemu_binary = utils_path.find_command('kvm')
-            logging.debug('Found %s', qemu_binary)
+        QEMU_BIN_NAMES = ['kvm', 'qemu-kvm', 'qemu-system-ppc64',
+                          'qemu_system', 'qemu-system-x86',
+                          'qemu-system-%s' % (ARCH)]
```

Comment:
I suppose 'kvm' should be the last one to try, so at the end of the list.

Reply:
`qemu-kvm` in rhel based systems and `kvm` in ubuntu based systems executes qemu-system-$ARCH with kvm enabled,
```shell
#!/bin/sh
exec qemu-system-$ARCH -enable-kvm "$@"
```
so we should be use them right if it is available ?

Reply:
@balamuruhans Yes, it will work well with current implementation, I just thought that qemu-kvm and other bin name have higher change to be used in our project. If put them behind `kvm`, every time it will go through kvm first(which waste a bit time). I think that original logic is better, it assumes that mostly we will have `qemu-kvm`, we check `kvm` only when there is no `qemu-kvm`.
Having said that, I have no strong opinion on this, it's functional good.

Reply:
I have changed the order based on your suggestion, please check if it is fine.